share/mk/bsd.cpu.mk

```diff
@@ -343,11 +342,12 @@ CFLAGS+=	-Werror=implicit-function-declaration
 # Turn off deprecated warnings
 # XXXBD: is this still needed?
 CFLAGS+=	-Wno-deprecated-declarations
```

Comment:
```suggestion
```
I think this was only needed for deprecated `__capability` placement and I (think I) fixed all of those ~2 years ago.

Reply:
It's under `mips*c*` so I'd hope so...

Reply:
Done.